projects/storefrontapp-e2e-cypress/cypress/integration/mobile/checkout-flow-mobile.e2e-spec.ts

```diff
@@ -26,10 +26,6 @@ context(`${formats.mobile.width + 1}p resolution - Big happy path`, () => {
     waitForHomePage();
 
     checkout.registerUser();
```

Comment:
Add  `verifyGlobalMessageAfterRegistration();` to avoid cancelling register requests by cypress